src/js/contentUtils.js

```diff
@@ -594,6 +594,16 @@ async function processJSWithSrc(script, origin, version) {
     if (sourceURLIndex == 0) {
       sourceText = '';
     } else if (sourceURLIndex > 0) {
+      // check to ensure there are no popups via MITM attack after //# sourceURL - check string contents after sourceURLIndex
+      const afterSourceURL = sourceText.slice(
+        sourceURLIndex,
+        sourceText.length
+      );
+      if (afterSourceURL.includes('alert')) {
+        return {
+          valid: false,
+        };
+      }
```

Comment:
We chatted about this over work chat. We need to remove this and beef up the detection

Reply:
added more url detection, including checking to ensure that the source is coming from fbcdn. Are there any source files that may not come from "static.xx.fbcdn.net"?